agent/php_stacked_segment.h

```diff
@@ -256,6 +256,8 @@
 // clang-format on
 
 #include "php_agent.h"
+#if ZEND_MODULE_API_NO < ZEND_8_0_X_API_NO \
+    || defined OVERWRITE_ZEND_EXECUTE_DATA /* not OAPI */
```

Comment:
This conditional compilation ensures that `nr_php_stacked_segment_*` API is only used by the agent that uses `zend_execute` hook and is not used by the agent using Observer API to hook into Zend Engine.